authorization/install.go

```diff
@@ -0,0 +1,26 @@
+package apps
+
+import (
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+
+	authorizationv1 "github.com/openshift/api/authorization/v1"
+)
+
+const (
+	GroupName = "authorization.openshift.io"
+)
+
+var (
+	schemeBuilder = runtime.NewSchemeBuilder(authorizationv1.Install)
+	// Install is a function which adds every version of this group to a scheme
+	Install = schemeBuilder.AddToScheme
```

Comment:
So at the version level we keep AddToScheme and at the group level we use `Install`.  I think I can live with that as it break less old code (in the upstream 3rdparty context).

Reply:
> So at the version level we keep AddToScheme and at the group level we use Install. I think I can live with that as it break less old code (in the upstream 3rdparty context).

We should transition away from AddToScheme